showcase.json

```diff
@@ -2998,8 +2998,28 @@
     "youtube": "https://www.youtube.com/@omnianetwork2947",
     "submittableId": "43839848"
   },
-
-  {
+    {
+    "name": "Flower Power DAO",
+    "description": "An NFT Trilogy and community-curated garden inspired by the French Artist, Ludo.",
+    "website": "https://fpdao.app",
+    "logo": "/img/showcase/fpdao_logo.webp",
+    "screenshots": [
+      "/img/showcase/fpdao_screenshot.webp"
+    ],
+    "video": "",
+    "display": "Normal",
+    "id": "fpdao",
+    "oneLiner": "The story of blockchain, told in Art.",
+    "stats": "",
+    "tags": [
+      "NFT",
+      "DAO"
+    ],
+    "usesInternetIdentity": true,
+    "github": "https://github.com/flowerpowerdao/",
+    "twitter": "https://twitter.com/btcflower",
```

Comment:
Can you please remove the comma at the end of the Twitter key value? (It expects another key value if you have a comma at the end)